packages/editor/src/components/post-title/index.native.js

```diff
@@ -100,7 +100,8 @@ class PostTitle extends Component {
 					} }
 					placeholder={ decodedPlaceholder }
 					value={ title }
-					onSplit={ this.props.onEnterPress }
+					onSplit={ () => { } }
```

Comment:
Do we need to set the `onSplit` to something? Can we just not set the prop at all?

Reply:
> https://github.com/WordPress/gutenberg/pull/15944#issuecomment-498146491
>
>  https://github.com/WordPress/gutenberg/pull/15944#issuecomment-498166693

Ah, I see 😿